ts/src/xt.ts

```diff
@@ -704,7 +704,35 @@ export default class xt extends Exchange {
          * @param {object} params extra parameters specific to the xt api endpoint
          * @returns {object} an associative dictionary of currencies
          */
-        const response = await this.publicSpotGetWalletSupportCurrency (params);
+        const [ chainsRequest, currenciesRequest ] = [ this.publicSpotGetWalletSupportCurrency (params), this.publicSpotGetCurrencies (params) ];
```

Comment:
this is an unconventional pattern. Usually, we do `const promisesRaw = (); const promises = Pormise.all(promisesRaw)`

Reply:
ah, I was pushing at this moment, you made this a second ago. tnx